launcher/ui/download.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, parent, fsgs):
         hori_layout.add(fsui.ImageView(self, fsui.Image(
             "launcher:res/16/folder.png")))
 
+        FSGSDirectories.ensure_dir_exists(FSGSDirectories.DOWNLOAD_DIR)
```

Comment:
this could our generic function to be invoked whenever we want to be sure that specific dir still exists. Could also be decorator or used with a `with ensure_dir_exists():`, why not?